docs/theory/statistics.rst

```diff
@@ -2,9 +2,84 @@
 Statistical issues
 ==================
 
-This document addresses statistical questions to do with Maud, such as:
+This document addresses statistical questions to do with Maud. Specifically, it
+sets out what Maud does from a statistical point of view, what statistical
+assumptions it makes and how these assumptions can be evaluated.
 
- - How does Maud frame the task of learning from multi-omics data as a
-   statistical inference problem?
+Bayesian inference for metabolic networks
+=========================================
 
- - What are some statistical assumptions that Maud makes?
+In general Bayesian inference offers two ways of expressing information
+probabilistically: 'prior' probability distributions over possible values of
+unknown parameters and measurement models or 'likelihood functions' that
+noisily relate parameter values to measurements. Together these determine a
+posterior distribution that synthesises the two sources of information and can
+be used for statistical inference.
+
+Maud's parameters are as follows:
+
+- thermodynamic parameters (the formation energy for each compound in the
+  network)
+- kinetic parameters governing the behaviour of each reaction under fixed
+  conditions
+- concentrations of unbalanced metabolites (i.e. metabolites whose
+  concentrations can be consumed or produced when the system is in a steady
+  state)
+- concentrations of enzymes
+
+The prior distributions for formation energies come from an upstream analysis
+involving in vitro measurements and inference about the composition of each
+compound. Kinetic parameter priors are inferred from online databases using the
+'respectful modelling' paradigm. Priors for concentrations of unbalanced
+metabolites and enzyme concentrations are determined respectively by
+metabolomics and proteomics measurements where these are available and by
+reasonable physical limits otherwise.
```

Comment:
We've changed it now haven't we? Now the priors for unbalanced metabolites and enzyme concentrations are just physiologically reasonable approximations, which could be tightened if needed.